src/renderer/components/basic.lua

```diff
@@ -79,24 +86,35 @@ function basicTextComponent:render(surf, position, styles, resolver)
       surf:drawSurfaceSmall(img, position.left + math.floor((position.width - rightPad - img.width / 2) / 2), cY)
     end
   elseif styles.content then
-    if styles["font-size"] == "2em" then
+    
+    if styles["font-size"] == "2rem" then
```

Comment:
We don't want to compare just to 2em, I know this is bc of #6, but I think we should still handle it correctly here, and just add the actual rendering functionality then.